accounts-db/src/stake_rewards.rs

```diff
@@ -119,3 +129,41 @@ impl StakeReward {
         self.stake_account.checked_add_lamports(amount).unwrap();
     }
 }
+
+#[cfg(feature = "dev-context-only-utils")]
+fn create_stake_account(
```

Comment:
We've had to duplicate a function across accounts-db and runtime in the past, so I'm fine with it here